src/Migration/Destinations/Appwrite.php

```
@@ -989,45 +953,46 @@ protected function createRow(Row $resource, bool $isLast): bool
                 /**
                  * This is in case an attribute was deleted from Appwrite attributes collection but was not deleted from the table
                  * When creating an archive we select * which will include orphan attribute from the schema
                  */
-                foreach ($this->rowBuffer as $row) {
-                    foreach ($row as $key => $value) {
-                        if (\str_starts_with($key, '$')) {
-                            continue;
-                        }
+                if ($dbForDatabases->getAdapter()->getSupportForAttributes()) {
+                    foreach ($this->rowBuffer as $row) {
+                        foreach ($row as $key => $value) {
```

PHPMD (2.15.0)

Avoid unused local variables such as '$value'. (undefined)

(UnusedLocalVariable)